clients/src/main/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImpl.java

```diff
@@ -997,11 +985,13 @@ void markReconciliationCompleted() {
      *     </li>
      * </ol>
      */
-    private void resolveMetadataForUnresolvedAssignment() {
-        assignmentReadyToReconcile.clear();
+    private SortedSet<TopicIdPartition> resolveMetadataForTargetAssignment() {
+        final SortedSet<TopicIdPartition> assignmentReadyToReconcile = new TreeSet<>(TOPIC_ID_PARTITION_COMPARATOR);
+        final HashMap<Uuid, SortedSet<Integer>> unresolved = new HashMap<>(currentTargetAssignment);
+
         // Try to resolve topic names from metadata cache or subscription cache, and move
         // assignments from the "unresolved" collection, to the "readyToReconcile" one.
```

Comment:
Let's update this comment now that we don't have the "unresolved" and "readyToReconcile" collections exactly in that shape anymore.

Reply:
I only updated the variable name `assignmentReadyToReconcile`. Otherwise, I think the comment is still valid.

I personally prefer to just have "why" comments and not "what" comments, so I'd be open to dropping the comment as well.

Reply:
Dropping it sounds good to me, agree with your what vs why view on this.